cmds/admin_server/storage/storage.go

```diff
@@ -0,0 +1,26 @@
+package storage
+
+import "time"
+
+type Storage interface {
+	StoreLog(Log) error
+	GetLogs(SearchOpt) ([]Log, error)
+
+	Close() error
+}
+
+// Log defines the basic log info pushed by the server
+type Log struct {
+	JobID    uint64    `json:"jobID"`
+	LogData  string    `json:"logData"`
+	Date     time.Time `json:"date"`
+	LogLevel string    `json:"logLevel"`
+}
+
+// SearchOpt defines the different options to filter with
```

Comment:
```suggestion
// SearchQuery defines the different options to filter with
```

or just Query, and remove the Query string; was that supposed to be text search? if so, rename to Text?